<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportExportGadgetWidget</class>
 <widget class="QWidget" name="ImportExportGadgetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBoxItems">
       <property name="title">
        <string>Items</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxGeneral">
          <property name="text">
           <string>General (Workspace, Key-Bindings)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxAllGadgets">
          <property name="text">
           <string>All Gadgets</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPlugins">
          <property name="text">
           <string>Plugins</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="helpButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../coreplugin/core.qrc">
           <normaloff>:/core/images/helpicon.svg</normaloff>:/core/images/helpicon.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="toolTip">
          <string>Export the GCS settings selected in the checkboxes above.</string>
         </property>
         <property name="text">
          <string>Export...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="toolTip">
          <string>Import settings from the config file, only for the items checked above.</string>
         </property>
         <property name="text">
          <string>Import...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="toolTip">
          <string>Resets your GCS configuration to its default configuration.</string>
         </property>
         <property name="text">
          <string>Reset Config</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../coreplugin/core.qrc"/>
 </resources>
 <connections/>
</ui>
